<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="widgetLayout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>705</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QGroupBox" name="groupBoxLaunchNew">
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Reset simulation</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Initial condition</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QComboBox" name="comboBoxInitialCondition">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Dual blob</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Dual galaxy</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Single galaxy</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Grid</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Moving grid</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Blobs</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Bodies</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="2">
                  <widget class="QSpinBox" name="spinBoxParticleCount">
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="minimum">
                    <number>2</number>
                   </property>
                   <property name="maximum">
                    <number>100000</number>
                   </property>
                   <property name="value">
                    <number>20000</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="pushButtonLaunch">
                   <property name="text">
                    <string>Reset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="2">
                  <widget class="QPushButton" name="pushButtonPause">
                   <property name="text">
                    <string>Pause</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Physics &amp;&amp; controls</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="4" column="0" colspan="2">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Softening factor</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QSlider" name="horizontalSliderMouseSensitivity">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>99</number>
                   </property>
                   <property name="value">
                    <number>20</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Mouse sensitivity</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Power parameter</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxSoftening">
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>0.000100000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.001000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.005000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0" colspan="2">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Time step</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxTimeStep">
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>0.000100000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.001000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.002000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="2">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Gravitational constant</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxGravityConst">
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>0.000100000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.001000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.001000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QSlider" name="horizontalSliderPower">
                   <property name="maximum">
                    <number>30</number>
                   </property>
                   <property name="value">
                    <number>15</number>
                   </property>
                   <property name="tracking">
                    <bool>true</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::NoTicks</enum>
                   </property>
                   <property name="tickInterval">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Graphics</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="2" column="1">
                  <widget class="QSlider" name="horizontalSliderBloomStrength">
                   <property name="value">
                    <number>45</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Particle size</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSlider" name="horizontalSliderGamma">
                   <property name="value">
                    <number>30</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSlider" name="horizontalSliderParticleSize">
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="sliderPosition">
                    <number>20</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QSlider" name="horizontalSliderExposure">
                   <property name="value">
                    <number>40</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSlider" name="horizontalSliderBloomExtent">
                   <property name="value">
                    <number>15</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Bloom strength</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Tone map mode</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QComboBox" name="comboBoxToneMappingMode">
                   <property name="currentIndex">
                    <number>1</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Reinhard</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Exposure</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Off</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Bloom extent</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Exposure</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Gamma</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0" colspan="2">
                  <widget class="QPushButton" name="pushButtonScreenshot">
                   <property name="text">
                    <string>Screenshot</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonHelp">
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Help</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
